unit-testing-frameworks/test/specs/add.spec.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+const CalculatorEntity = require("../../app/calculatorEntity");
+const {expect} = require("chai");
+
+describe("method \"add()\": positive scenarios", function() {
+    let calculator;
+
+    beforeEach(() => calculator = new CalculatorEntity());
+    afterEach(() => calculator = null);
+
+    it("must return sum of inputted arguments if they are integers",
+        () => expect(calculator.add(-43, 11, 35)).to.be.equal(3));
+
+    it("must return sum of inputted arguments if they include integers"
+                                             +" and floating-point numbers",
+        () => expect(calculator.add(5.5, 11, 15.2, -1.3)).to.be.equal(30.4));
```

Comment:
you don't need to verify how JS works.

Reply:
Thank you! Ok, I'll take that into account for my future tasks.